README.md

```diff
@@ -1,34 +1,65 @@
 # SSA
 
+![Core library build status](https://github.com/openncompl/ssa/actions/workflows/ci.yml/badge.svg)
+
+
 Theory of static single assignment development in the Lean
 proof assistant.
 
 - [Link to generated documentation](https://opencompl.github.io/ssa/)
 
+#### Installation
+
+- First, setup the Lean toolchain with [elan](https://github.com/leanprover/elan?tab=readme-ov-file#installation).
+- Next, run:
+
+```
+git clone https://github.com/opencompl/ssa.git && cd ssa && lake exe cache get && lake build
+```
+
```

Comment:
should we have some text about how to prove rewrites correct, etc?

Reply:
@goens I'd suggest we do this in a separate PR where we point people to the PaperExamples subdirectory, perhaps?